nssp_demo/forecast_state.py

```diff
@@ -49,26 +49,41 @@ def main(
     disease: str,
     report_date: str,
     state: str,
-    nssp_data_dir: Path | str,
+    facility_level_nssp_data_dir: Path | str,
+    state_level_nssp_data_dir: Path | str,
     param_data_dir: Path | str,
     output_data_dir: Path | str,
     n_training_days: int,
     n_forecast_days: int,
     n_chains: int,
     n_warmup: int,
     n_samples: int,
-    last_training_date: str,
+    last_training_date: str = None,
 ):
     logging.basicConfig(level=logging.INFO)
     logger = logging.getLogger(__name__)
 
+    available_facility_level_reports = [
+        datetime.strptime(f.stem, "%Y-%m-%d").date()
+        for f in Path(facility_level_nssp_data_dir).glob("*.parquet")
```

Comment:
Could use a more specific glob pattern to ensure ` datetime.strptime` will not fail by being called on non-compliant files (though that shouldn't happen).

Reply:
I think we can leave this for now, but in fddcbfe I have created a `get_available_reports` helper function that we could modify in the future to have different (potentially stricter) logic.